webrtc/libjingle/xmpp/boshsocket.cc

```diff
@@ -155,7 +156,15 @@ namespace buzz {
 #ifdef FEATURE_ENABLE_SSL
     if (tls_ != buzz::TLS_DISABLED) {
       first_socket = rtc::SSLAdapter::Create(first_socket);
+// For non-production builds, relax certificate requirements. This will allow the use
+// of a debugging proxy, for example.
+#ifndef _PRODUCT
```

Comment:
I meant that we probably aren't compiling WebRTC with PSY_PRODUCT set, today, but we can change that.